build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -229,6 +225,12 @@ fun configureTests() {
             println("Test distribution has been enabled for $testName")
             distribution {
                 enabled.set(true)
+
+
+                // Dogfooding TD against ge-experiment until GE 2021.1 is available on e.grdev.net and ge.gradle.org (and the new TD Gradle plugin version 2.0 is accepted)
+                this as TestDistributionExtensionInternal
+                server.set(uri("https://ge-experiment.grdev.net"))
```

Comment:
I think this would be easier to read if written like:
`(this as TestDistributionExtensionInternal).server.set(uri("https://ge-experiment.grdev.net"))`

Reply:
Make sense. Fixed.